{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -69,8 +81,13 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
                 logger.info(f'Deleting artifact at {str(path)}.')
                 path.unlink()
 
-        build_all_artifacts(output_dir, verbose)
-
+        if running_from_cluster():
+            # parallel build when on cluster
+            build_all_artifacts(output_dir, verbose)
+        else:
+            # serial build when not on cluster
+            for loc in project_globals.LOCATIONS:
```

Comment:
This will prompt the user before each location if there is an existing artifact at the location. It would be better either to prompt only once or prompt once for each location, but to do them all up front.

Reply:
Yes, I stepped back and reimplemented things. See if you think is is better.